### ISSUE TITLE
Segmentation results

### ISSUE THREAD

**Issue by Wenhui-Zhang-5** [Usman-Rafique/Probabilistic_UNet#1]
Hi 

  Thank you for your code. They are useful and helpful. I have several quick questions. The final out_samples are not 0-1 segmentation. Are they softmax outputs? 

Best
Stella

**Comment by Usman-Rafique** [comment]
Hi. Thank you for your nice words. Sorry for the late reply, I missed notification about this issue.

You are right, the network output is not a 0-1 segmentation mask. The `sample` returned by the model contains logits. The model output should first be converted to probability by applying a sigmoid function, as done here: https://github.com/Usman-Rafique/Probabilistic_UNet/blob/master/visualize.py#L64. Finally, if you need a binary mask, you should apply a threshold on these probability scores.

I am closing this issue for now. If you have any more questions, please feel free to ask, and re-open the issue if you want.

Thanks
